index.html

```diff
@@ -299,32 +299,37 @@
 
 
       function translate(node, info) {
-        if((node.nodeType === 1 && node.hasAttribute('data-ly-locked')) || (node.nodeType === 3 && node.parentNode && node.parentNode.hasAttribute('data-ly-locked'))) {
+        const nodeType = node.nodeType;
+        const nodeName = node.nodeName;
```

Comment:
Both `node.nodeType` and `node.nodeName` are accessed multiple times inside this function so it's worth caching their values in a variable for faster lookup.